packages/database/migrations/0001_create_object_item.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE `object_item` (
+  `object_hash` varchar(32) NOT NULL,
```

Comment:
We could name this `id` (to be in sync with `id` in table `object_list`)?

Then it's an implementation detail that we use hashes of IRIs for these IDs.